### DISCUSSION TITLE
Failing when following Cypress Docs Google-Authentication - cy.task(db:seed)  'task' not registered

### DISCUSSION THREAD

**Discussion by larryfast** [cypress-io/cypress#29573] [Questions and Help]
I'm working my way through Google Authentication instructions found here:
https://docs.cypress.io/guides/end-to-end-testing/google-authentication

When I attempt to run the Test Login code I get the following error
```
cy.task('db:seed') failed with the following error:
The 'task' event has not been registered in the setupNodeEvents method. You must register it before using cy.task()
```
I don't see any references to setupNodeEvents cy.task() or db:seed elsewhere on the page.
I tried 
- commenting out the line - different long winded error
- add a dummy on(task, ...   entry in the setupNodeEvents() block in cypress/cypress.config.js - no change
- add setupNodeEvents in cypress.config.ts - syntax error

More details:
The code block did not include any info about placement so I created a file playgroundAuth.cy.ts in my test suite folder. I also tried changing it to .js but got the same error.  Here's the code block from the page:
```
describe('Google', function () {
  beforeEach(function () {
    cy.task('db:seed')
    cy.loginByGoogleApi()
  })

  it('shows onboarding', function () {
    cy.contains('Get Started').should('be.visible')
  })
})
```

**Comment by portyu9** [comment]
Hi @larryfast 

`db:seed` is **not a built-in Cypress task**. It was specific to the example application used by that authentication guide.

`cy.task('db:seed')` only works if your `cypress.config.js` / `cypress.config.ts` registers a task with that exact name inside `setupNodeEvents`.

So you have two options:

- If your application does not need database seeding, remove `cy.task('db:seed')`.
- If it does, implement your own `db:seed` task and register it in `e2e.setupNodeEvents`.

Also make sure you are editing the project's root `cypress.config.*`, not a config file under the `cypress/` directory unless you explicitly launch Cypress with that config.

Please mark this answer as accepted if it helped, thank you.
